src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private String nameColorGenerate = "";
+    private int magicNumber = 100;

```

Comment:
```suggestion
    private final static int MAX_VALUE = 100;

```

магіческим числом називають те, що не зрозуміло по коду, що це за число :)
тобто дивишься в код, а там наприклад 100, чому 100, що за 100
якщо винести в константу та дати назву, більш зрозуміло становиться і легше читати код